application/clicommands/ScanCommand.php

```diff
@@ -58,9 +70,44 @@ public function indexAction()
                 $name
             );
 
+            $sinceLastScan = $this->processThresholdToDateTiime($this->params->get('since-last-scan'));
```

Comment:
Why not processing the threshold to a date time only if it''s not `null`? Function `processThresholdToDateTime()` can expect then not nullable argument.